repl/commands.go

```diff
@@ -84,16 +87,27 @@ func Parse(line string) (cmd string, args []string, expr string, err error) {
 		m := delRe.FindStringSubmatch(line)
 		if m == nil {
 			err = PromptError("invalid delete statement")
+			return
 		}
 		if i := subexpByName(delRe, m, "ident"); i != nil {
 			args = append(args, *i)
 		}
+	case "declare":
+		m := declRe.FindStringSubmatch(line)
+		if m == nil {
+			err = PromptError("invalid declare statement")
```

Comment:
Here and elsewhere: it's helpful to give a hint about a valid statement in addition to mentioning the current input is invalid.

Reply:
Done